core/app/models/spree/payment_method.rb

```diff
@@ -29,13 +27,12 @@ def payment_source_class
     def self.available(display_on = 'both')
       all.select do |p|
```

Comment:
Just a user account I didn't mean to comment with. It's used to signup for services such as circleci under a spree account rather than my own.